api-docs/list-clients.md

```diff
@@ -4,15 +4,15 @@ API is provided to query a list of clients from the database.
 
 ## Request
 
-GET /api/clients?firstName=Mario&lastName=Luigi&zip=84107&page=1
+GET /api/clients?name=Mario%20Luigi&zip=84107&page=1
```

Comment:
Just searching by `name` will be an easier UX for the frontend, so we can let the user type in "Juan Rodriguez" without having to change to two different inputs to search for it.

Reply:
I agree that will be better..